communication/slack-config/channels.yaml

```diff
@@ -147,7 +147,6 @@ channels:
   - name: krew
   - name: krex
   - name: kube-aws
-  - name: kube-deploy
```

Comment:
the channel is inactive or has out-of-context messages.

i could not find the steps for achiving a slack channel in:
https://github.com/kubernetes/community/blob/master/communication/slack-guidelines.md

so maybe one has to ask in `#slack-admins`.

Reply:
yet CI gave this error here:
```
2020/05/20 20:42:04 This configuration cannot be applied against the current reality:
2020/05/20 20:42:04 Error 1: channel kube-deploy (C992D6ZPC) not referenced in config.
2020/05/20 20:42:04 In dry run mode so taking no action, but this is what we would've done:
2020/05/20 20:42:04 Nothing to do.
```

Reply:
@neolit123 It needs  `archived: true` :)

Example:

https://github.com/kubernetes/community/blob/32c4a99bce3a9be2a0a3a77cdba406d8f53f1076/communication/slack-config/channels.yaml#L42

Reply:
thanks, updated.